src/main/java/group1/mips_simulator/SimApplication.java

```diff
@@ -7,10 +7,10 @@
 
 import java.io.IOException;
 
-public class HelloApplication extends Application {
+public class SimApplication extends Application {
```

Comment:
So because you're making the change here, and in your other MR, there may be a merge conflict. I think in this case it's ok (because you're changing a small thing and it's exactly the same between the two).

But generally I'd recommend avoiding this kind of duplicated work going forward.

Reply:
makes sense

Reply:
also i dont think my directory changes actually got pushed through, i may need help with that